src/rez/rezconfig.py

```diff
@@ -366,10 +366,18 @@
 # scripts (such as .bashrc). If False, package commands are sourced after.
 package_commands_sourced_first = True
 
+# Defines how the standard_system_paths configuration acts on the $PATH:
+# - "replace":           Replace $PATH if standard_system_paths is set
+# - "append":            Append standard_system_paths to determined $PATH
```

Comment:
This is confusing, I think you mean:

```
Append 'standard_system_path' to automatically determined system paths
```

right?

Reply:
Ok ignore that, I think I have an idea on how to make this clearer. It's jut inherently a little hard to explain I think.